app/api/chemotion/chem_spectra_api.rb

```diff
@@ -208,7 +208,8 @@ def raw_file(att)
               protocol = nmrium_uri.scheme
               host = nmrium_uri.host
               port = nmrium_uri.port == 443 ? '' : nmrium_uri.port
-              { protocol: protocol, host: host, port: port }
+              nmrium_origin = port == '' ? "#{protocol}://#{host}" : "#{protocol}://#{host}:#{port}"
```

Comment:
I want to do it in the backend, the frontend only needs to get the values and compare. The `nmrium_origin` is used in the frontend to fill the message from nmrium wrapper.
The comparison of port value is to make sure we don't use port 443 in the past.
But as you suggested, we're using `nmriumwrapper` in the `spectra.yml` instead of separated values, I leave it as to cover the older config file.
I am open to changing this, but is it necessary or not?